addons/vehiclelock/functions/fnc_moduleSync.sqf

```diff
@@ -28,7 +28,8 @@ if !(_activated) exitWith {WARNING("Vehicle Lock Sync Module - placed but not ac
     params ["_syncedObjects"];
 
     private _listOfVehicles = _syncedObjects select {
-        (_x isKindOf "Car") || {(_x isKindOf "Tank") || {_x isKindOf "Helicopter"}}
+        #define CLASSNAMES ["Car", "Tank", "Air"]
```

Comment:
macro is useless here

Reply:
Commas break the args in a macro, because the preprocessor cannot handle the parentheses syntax. You'd have to use the ARR_3 macro which is ugly.